src/app/components/AmpExperiment/index.tsx

```diff
@@ -0,0 +1,46 @@
+/** @jsx jsx */
+/* @jsxFrag React.Fragment */
+import { jsx } from '@emotion/react';
+import React from 'react';
+import { Helmet } from 'react-helmet';
+
+type AmpExperimentConfig = {
+  [key: string]: {
+    sticky?: boolean;
+    consentNotificationId?: string;
+    variants: {
+      [key: string]: number; // variantName: traffic allocation %
```

Comment:
**Suggestion** You could extract this type into some variables and so remove the need for the comment as you can define the parts `trafficAllocation: number`, `variants: Variants`